types/gif.js/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for gif.js 0.2
+// Project: https://github.com/jnordberg/gif.js#readme
+// Definitions by: Carlos Precioso <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { EventEmitter } from 'events';
+
+declare namespace GIF {
```

Comment:
```suggestion
export as namespace GIF;

declare namespace GIF {
```
so it looks this is an UMD module, with 'GIF' exported:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#optional-global-usage
(read from script atteched to the page as per README)